internal/fs/inode/dir.go

```diff
@@ -670,6 +681,9 @@ func (d *dirInode) ReadEntries(
 		}
 		entries = append(entries, entry)
 	}
+
+	tempTime := time.Now()
```

Comment:
should we use cacheClock ? that would allow you to write unit tests as well without adding time.Sleep in the code.

Reply:
+1